2016/presentations/Python_for_Networking_Devices/text.md

```diff
@@ -0,0 +1,139 @@
+#Python for Networking Devices - Elisa Jasinska
+
+##Introduction
+
+The Internet is a series of tubes and at the end of those tubes are: networking devices! To form the Internet as we
+know it, each provider network has to be managed, maintained and interconnected. Traditional Network Engineering is
+moving more and more towards automated device and service management, a task often performed in Python due to the
+availability of many useful libraries. We will walk you though common tasks in Network Engineering and introduce a
+number of Python libraries that are helpful in accessing and managing networking equipment.
+
+##Network Device Access
+
+Traditionally networking devices are managed via their command line interface (CLI). The CLI is accessible though
+various forms of transport. Originally Telnet has been used, later on devices started to support SSH. SSH provides
+additional security mechanisms over Telnet, but its usability doesn't differ much, it's still a CLI.
+
+Configuration commands are send via the CLI, line by line, and enable or disable specific functionality on the device.
+For example, to configure a BGP neighbor on your router, your might have to add a config block similar to this one:
```

Comment:
'might have to add a config block'
This is suggesting that we are going to send config through CLI. I would change that to 'your config may need configuration like' (this will suggest that next lines are going to make this entry exist).